### PR TITLE
assorted cleanups

### PR DESCRIPTION
**Pull request by oxij**
###### Motivation for this change

Cleanups.

###### Things done

- [X] Trivial.
- [X] Works.
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Review by 7c6f434c — APPROVED** [review]
Read the changes. LGTM, not merging immediately to give time for feedback from people using mainline NixOS.

File: nixos/modules/services/misc/dysnomia.nix [hunk 1]
```diff
@@ -72,59 +72,59 @@ let
 in
 {
   options = {
-    dysnomia = {
-      
+    services.dysnomia = {
```
Comment (7c6f434c) [inline nixos/modules/services/misc/dysnomia.nix]: Maybe add `dysnomia` as an alias with a rename warning?

**Comment by oxij** [comment]
> Maybe add `dysnomia` as an alias with a rename warning?

I did. See `rename.nix`.

**Comment by 7c6f434c** [comment]
Ah right, sorry.

**Comment by oxij** [comment]
ping?

**Comment by 7c6f434c** [comment]
OK, looks like nobody cares to request any changes.

**Pull request merged**